org.code.toboggan.network/src/org/code/toboggan/network/NetworkActivator.java

```diff
@@ -17,6 +18,8 @@ static BundleContext getContext() {
 	 */
 	public void start(BundleContext bundleContext) throws Exception {
 		NetworkActivator.context = bundleContext;
+		// Start WS connection when this plugin is activated
+		WSService.getWSManager().connect();
```

Comment:
In our old pluginmanager code, we have logic that listens for the onConnect event of the websocket, and this worked since connect was called after the listeners were hooked up. However, doing this here, we are connecting before (or at the same time as) when the listeners are set up. Is this a problem?